components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/device/dao/DeviceFlowDAOImpl.java

```diff
@@ -375,12 +549,12 @@ public void setCallbackURI(String clientId, String callbackUri) throws IdentityO
      * @param connection  Database connection.
      * @throws IdentityOAuth2Exception Error while storing parameters.
      */
+    @Deprecated
```

Comment:
Why is the private method deprecated. let's remove the method and update the places it is being used accordingly